Assets/LazurFiringAnimator.cs

```diff
@@ -28,6 +30,8 @@ public void EndFiring()
 
     public override void OnFire(GunStats stats)
     {
+        if (IsDiabledInReload && stats.Ammo <= 1)
```

Comment:
This should actually be in all the firinganimators I think (at least for coil animator as well since it also is possible to start firing that one during reload)

Reply:
It could, but I'm very worried about unforseen regressions on some of the implementations. I'll add it to the coil manually for now and I'll put up an refactor issue we can do after the update so we have the time properly test before this is released 👍